### PR TITLE
GitBook: [dx-to-add-own-connector_suggestion] 5 pages and 5 assets mo…

### PR DESCRIPTION
**Pull request by johnlafleur**
## What
If a developer wants to add their own connectors and use them without contributing to the codebase, there is no simple article to reply to all their questions. This is a suggestion of a new documentation article to address this use case. 

This is something we discussed with Sherif in our 1:1 (that doesn't mean he has seen the suggestion beforehand!).

### CODE REVIEW

File: docs/contributing/contributing-to-airbyte/README.md [hunk 1]
```diff
@@ -4,45 +4,92 @@ description: 'We love contributions to Airbyte, big or small.'
 
 # Contributing to Airbyte
 
-We hope you are enjoying Airbyte and we look forward to hearing your feedback. Whether you want to raise a bug report, a feature/integration request or just request a correction to something, we are eager to get input from our community.
+_First_: if you feel insecure about how to start contributing, feel free to ask us on [Slack](https://slack.airbyte.io) in the \#dev or even \#general channel. You can also just go ahead with your contribution and we'll give you feedback. Don't worry,  the worst that can happen is that you'll be politely asked to change something! We appreciate any contributions, and we don't want a wall of rules to stand in the way of that.
```
Comment (michel-tricot) [inline docs/contributing/contributing-to-airbyte/README.md:7]: I wouldn't use the word insecure, it's a bit too strong "if you don't know where to start" ?

File: docs/contributing/contributing-to-airbyte/README.md [hunk 2]
```diff
@@ -4,45 +4,92 @@ description: 'We love contributions to Airbyte, big or small.'
...
+* Help us in adding to the docs
+
+The contributing guide for docs can be found [here](https://docs.airbyte.io/contributing/contributing-to-airbyte/updating-documentation).
+
+### **Community content**
+
+Since we launched our [Tutorials page](http://airbyte.io/tutorials), we are happy about contributions:
+
+* Fix errors in existing learn tutorials
+* Add new tutorials \(please reach out to us if you have ideas to avoid duplicate word\)
```
Comment (michel-tricot) [inline docs/contributing/contributing-to-airbyte/README.md:66]: Typo: work

**Review by michel-tricot — APPROVED** [review]

**Comment by sherifnada** [comment]
@johnlafleur this seems to be the same as the other PR, or at least have a lot overlap? is this intentional? I left my review on the other one, LMK if you'd like it on this one as well.

**Comment by johnlafleur** [comment]
It seems it kept the changes from the other branch. But this one actually has a new page “Add your own connector” under the Integrations section. This PR is only about this page.

**Comment by johnlafleur** [comment]
I'm going to close this pull request and create a new one. Gitbook messed everything up with this one.

**Pull request closed without merging**